spec/cypress/support/step_definitions/accounts.js

```diff
@@ -42,3 +42,14 @@ Given(
     ]).as("userData");
   }
 );
+
+Given("Following Factory set up was run {string}", (factoryName) => {
+  cy.appFactories([["create", factoryName]]).as("factoryData");
+});
+
+Given(
+  "Following Factory set up was run {string} with trait {string}",
+  (factoryName, traitName) => {
+    cy.appFactories([["create", factoryName, traitName]]).as("factoryData");
+  }
+);
```

Comment:
Looks like I beat you to it here! 😁 

https://github.com/DFE-Digital/early-careers-framework/blob/develop/spec/cypress/support/step_definitions/factory-bot.js

Reply:
Done - reused your code